tfhe/src/core_crypto/entities/seeded_lwe_keyswitch_key.rs

```diff
@@ -420,3 +420,29 @@ impl<Scalar: UnsignedInteger, C: ContainerMut<Element = Scalar>> ContiguousEntit
     where
         Self: 'this;
 }
+
+impl<C: Container<Element = u64>> ParameterSetConformant for SeededLweKeyswitchKey<C> {
+    type ParameterSet = KeyswitchKeyConformanceParams;
+
+    fn is_conformant(&self, parameter_set: &Self::ParameterSet) -> bool {
+        let Self {
+            data,
+            decomp_base_log,
+            decomp_level_count,
+            output_lwe_size,
+            ciphertext_modulus,
+            compression_seed: _,
+        } = self;
+
+        *ciphertext_modulus == parameter_set.ciphertext_modulus
+            && data.container_len()
+                == parameter_set.input_lwe_dimension.0
+                    * lwe_keyswitch_key_input_key_element_encrypted_size(
+                        parameter_set.decomp_level_count,
+                        parameter_set.output_lwe_size,
+                    )
```

Comment:
that can't be right either, this should parameter_set.input_lwe_dimension.0 as we only store one body per encryption

Reply:
Fixed